File: Jour3/jour3.c
#include<stdio.h>

int main()
{
	float longueur,largeur;
	printf(" ============ Ce programme vous permet de calculer l'air et le perimetre d'un rectangle ============\n");
	printf("Donner la longeur de votre rectangle : ");
	scanf("%f",&longueur);
	//Attention aucun controle de la variable saisie

	printf("Donner la largeur de votre rectangle : ");
	scanf("%f",&largeur);
	//Attention aucun controle de la variable saisie	

	printf("Le perimetre vaut : %.1f et l'air vaut : %.1f",(longueur + largeur)*2,longueur*largeur);
	return 0;
}

File: Jour3/jour3.pas
program Calcul;
var
Longueur : real;
Largeur    : real;

begin
	writeln(' ============== Ce programme calcul l"aire et le perimetre d"un rectangle ==============');
write('Donner la longueur de votre rectangle : ' );
read(Longueur);
//Attention aucun control sur la valeur saisie

write('Donner la largeur de votre rectangle : ');
read(Largeur);
//Attention aucun control sur la valeur saisie

writeln('Le perimetre vaut : ',((Longueur + Largeur)*2):0:2,' et l"aire vaut : ', (Longueur * Largeur):0:2);
end.